src/App.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Created by ttnd on 27/12/16.
+ */
+import React from 'react';
+import Header from './Header';
+import Users from './Users';
+import Footer from './Footer';
+
+export default class App extends React.Component {
+  constructor() {
```

Comment:
Defining constructor is not mandatory on every component.
You can skip it if no logic is to be written in constructor